faq/templates/faq/faqadmin.html

```diff
@@ -0,0 +1,41 @@
+{% extends "web/base.html" %}
+{% load i18n %}
+
+{% block body %}
+    <div class="ui container">
+        <h1>
+            {% trans "Questions" %}
+            {% if perms.faq.add_question %}
+                <a href="{% url "FAQ-create" %}">
+                    <i class="green plus icon"></i>
+                </a>
+            {% endif %}
+        </h1>
+        <div class="ui relaxed divided list">
+            {% for question in questionlist %}
+                <div class="item">
+                    <div class="content">
+                        <div class="ui header">
+                            {{ question.title }}
+                            {% if perms.faq.change_question  %}
+                                <a href="{% url "FAQ-edit" pk=question.pk %}">
+                                    <i class="yellow pencil icon"></i>
+                                </a>
+                            {% endif %}
+
+                        {% if perms.faq.delete_question %}
+                            <a class="delete confirm"
+                               data-title="{% trans "Delete" %}"
+                               data-content="{% trans "Click to delete" %}."
+                               href="{% url 'FAQ-question-delete' question.id %}">
```

Comment:
Please use the `pk` property instead of the `id` field, as it's best practice (since it always refers to the model's primary key field, which could potentially be named something other than `id`) 😊 
```suggestion
                               href="{% url 'FAQ-question-delete' question.pk%}">
```